packages/qocamlbrowser/qocamlbrowser.0.2.10/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "kakadu.hafanana@gmail.com"
+homepage: "https://github.com/Kakadu/QOcamlBrowser_quick/"
+bug-reports: "https://github.com/Kakadu/QOcamlBrowser_quick/issues"
+authors: "Kakadu"
+dev-repo: "git://github.com/Kakadu/QOcamlBrowser_quick"
+
+build: [
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"] {os != "darwin" }
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]        {os != "darwin" }
+
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH ./configure"]               {os  = "darwin" }
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH make"]                      {os  = "darwin" }
+]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [
+  [make "uninstall"]
+]
+depends: [
+  "lablqml" {>= "0.5.1"}
+  "conf-qt" {>= "5.2.1"}
+  "ocamlbuild" {build}
+]
+available: (ocaml-version >= "4.07.0")
```

Comment:
any reasons why this restriction? I can't see any recent commit using 4.07 features

Reply:
Oops, I did not notice that 😱 
Maybe it was meant to be `<=`?

Reply:
I'll send a PR

Reply:
https://github.com/ocaml/opam-repository/pull/12384

Reply:
By some reason from https://github.com/ocaml/opam-repository/pull/12367#issuecomment-405916478 I thought that `Ident.name` fix introduces `>= 4.07` constraint. It seems that I was wrong...